applications/OptimizationApplication/custom_utilities/response/mass_response_utils.cpp

```diff
@@ -0,0 +1,271 @@
+//    |  /           |
```

Comment:
@sunethwarna I am convinced to change the MassReponse to the utilities. We lose consistency if we do so. Mass is a response and its calculations should be under MassResponse class not use utilities to do so. Anyway, we should talk about it. Maybe thereare  reasons.

Reply:
As we discussed, I moved the if blocks to the cpp level. Hence closing this comment.